{"name": "test_registration_with_invalid_data", "status": "passed", "start": 1721831573046, "stop": 1721831575088, "uuid": "f23d72fe-b8d2-49e2-9ee4-b3422490a50d", "historyId": "20abe179da2ce0e7e7aabd1e3acbd83e", "testCaseId": "20abe179da2ce0e7e7aabd1e3acbd83e", "fullName": "test.test_Registration.TestLogin#test_registration_with_invalid_data", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_Registration"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Raunak"}, {"name": "thread", "value": "33780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_Registration"}]}